tests/tools/onert_train/src/args.cc

```diff
@@ -224,18 +275,21 @@ void Args::Initialize(void)
     ("epoch", po::value<int>()->default_value(5)->notifier([&](const auto &v) { _epoch = v; }), "Epoch number (default: 5)")
     ("batch_size", po::value<int>()->default_value(32)->notifier([&](const auto &v) { _batch_size = v; }), "Batch size (default: 32)")
     ("learning_rate", po::value<float>()->default_value(0.001)->notifier([&](const auto &v) { _learning_rate = v; }), "Learning rate (default: 0.001)")
-    ("loss", po::value<int>()->default_value(1)->notifier([&] (const auto &v) { _loss_type = v; }),
-        "Loss type\n"
-        "1: MEAN_SQUARED_ERROR (default)\n"
-        "2: CATEGORICAL_CROSSENTROPY")
-    ("loss_reduction_type", po::value<int>()->default_value(1)->notifier([&] (const auto &v) { _loss_reduction_type = v; }),
-        "Loss Reduction type\n"
-        "1: SUM_OVER_BATCH_SIZE(default)\n"
-        "2: SUM")
-    ("optimizer", po::value<int>()->default_value(1)->notifier([&] (const auto &v) { _optimizer_type = v; }),
-      "Optimizer type\n"
-      "1: SGD (default)\n"
-      "2: Adam")
+    ("loss", po::value<int>()
+      ->default_value(default_loss)
+      ->notifier([&](const auto &v) {_loss_type = check_validation<NNFW_TRAIN_LOSS>(valid_loss ,v, "loss");}),
```

Comment:
It looks little difficult but so much better.

Let's get our goal to achieve simpler like in the future.
```cpp
    ("nnpackage", po::value<std::string>()->notifier(process_nnpackage), "NN Package file(directory) name")
    ("modelfile", po::value<std::string>()->notifier(process_modelfile), "NN Model filename")
```

My idea is making simpler below code
```cpp
// as-is
    ("loss_reduction_type", po::value<int>()
      ->default_value(default_loss_rdt)
      ->notifier([&] (const auto &v) {_loss_reduction_type = check_validation<NNFW_TRAIN_LOSS_REDUCTION>(valid_loss_rdt, v, "loss_reduction_type");}),
      gen_help_msg<NNFW_TRAIN_LOSS_REDUCTION>("loss_reduction_tye", valid_loss_rdt, default_loss_rdt).c_str()
    )

// to-be
    ("loss_reduction_type", po::value<int>()
      ->default_value(default_loss_rdt)
      ->notifier(process_loss_reduction_type),
      help_msg_loss_reduction_type()
    )
```

(I don't think the time is now. Just opinion. 😄)